backend/models/Comment.js

```diff
@@ -1,12 +1,33 @@
-import mongoose from 'mongoose';
+import mongoose from "mongoose";
 
 const commentSchema = new mongoose.Schema({
-  recipe: { type: mongoose.Schema.Types.ObjectId, ref: 'Recipe', required: true }, // Link to the related recipe
-  username: { type: String, required: true },     // User who made the comment
-  content: { type: String, required: true },
-  date: { type: Date, default: Date.now },
+  recipe: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "Recipe",
+    required: true,
+  }, 
+
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+  }, 
+  username: {
+    type: String,
+    required: true,
+  }, 
+
+  content: {
+    type: String,
+    required: true,
+  },
+
+  date: {
+    type: Date,
+    default: Date.now,
+  },
 });
 
-const Comment = mongoose.model('Comment', commentSchema);
+const Comment = mongoose.model("Comment", commentSchema);
 
-export default Comment;
\ No newline at end of file
+export default Comment;
```

Comment:
@haseebzaki-07 make sure the change in model does not change any other functionality

Reply:
@AlfiyaSiddique the model change does not affect any other functionality.